### PR TITLE
Tries to address all problems discussed on issue #157

### PR DESCRIPTION
**Pull request by coderReview**
Related to issue #157

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -163,6 +163,7 @@
     <config-file target="AndroidManifest.xml" parent="/manifest/application">
       <activity android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale"
           android:label="@string/app_name"
+          android:launchMode="singleTop"
```
Comment (bhariharan) [inline plugin.xml]: @coderReview Why are you adding ```singleTop``` here? I don't see any reason to launch this in a separate task.

Comment (coderReview) [inline plugin.xml]: Sorry. Forgot to remove that after testing. Will update the commit to remove that.

File: src/android/libs/SmartStore/build.gradle [hunk 2]
```diff
@@ -20,7 +20,7 @@ android {
       renderscript.srcDirs = ['src']
       res.srcDirs = ['res']
       assets.srcDirs = ['assets']
-      jniLibs.srcDir '../../external/sqlcipher/libs'
+      jniLibs.srcDir 'libs'
```
Comment (bhariharan) [inline src/android/libs/SmartStore/build.gradle]: I think I fixed this a while ago on ```unstable```. @coderReview Would you mind rebasing this pull request off the latest from ```unstable```. I'll run some tests locally after and then get it merged. Thanks!

Comment (coderReview) [inline src/android/libs/SmartStore/build.gradle]: @bhariharan  Doing it now.

**Pull request closed without merging**